.ci/check-for-new-tag.sh

```diff
@@ -15,7 +15,7 @@ ltag_commit=$(git rev-list -n 1 "$latest_tag")
 # if latest tag is a descendant of the latest tag commit, return pass
```

Comment:
It occurs to me that we're checking for the latest tag incorrectly here. Flagship thinks there's a new tag available, and there isn't.

What we should be doing is:
* Getting the hash value from the site's `composer.lock` file and making that the `latest_commit`
* Then getting the latest_tag from the `wri_sites` repo. I'm pretty sure the `git` commands here are running against the site repo, like `wriflagship` so you'll have to check out the wri_sites repo somewhere outside of the repo (just at `../` maybe?)
* Then running the rest of the commands from within the site folder.